src/v2/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -164,18 +165,24 @@ export class CollectionApp extends Component<CollectionAppProps> {
                 </ArtworkFilterContextProvider>
               </Box>
 
-              {collection.linkedCollections.length === 0 && (
-                <>
-                  <Separator mt={6} mb={3} />
-                  <Box mt="3">
-                    <RelatedCollectionsRail
-                      collections={collection.relatedCollections}
-                      title={collection.title}
-                      lazyLoadImages
-                    />
-                  </Box>
-                </>
-              )}
+              {/* HOTFIX FIXME: This rail was causing an error if included in SSR render
+                  pass and so it was deferred to the client.
+
+                  See: https://github.com/artsy/force/pull/6137
+              */}
+              {collection.linkedCollections.length === 0 &&
+                typeof window !== "undefined" && (
```

Comment:
Why does this need this line? (thought that might be implicit in the `LazyLoadComponent` and I don't see it above).

Reply:
I _believe_ without it we'll run into issues during the server render pass (can't find `window`). Copied from the artist page.